tools/spec-gen-sdk/package.json

```diff
@@ -5,7 +5,7 @@
     "email": "azsdkteam@microsoft.com",
     "url": "https://github.com/Azure/azure-sdk-tools"
   },
-  "version": "0.6.1",
+  "version": "0.6.2",
```

Comment:
This is a breaking change and can you bump the version to 0.7.0. Then you can validate the changes along with the runner changes in pipeline test run.